06/README.md

```diff
@@ -2,6 +2,8 @@ Now that you've written a function to pass a given test, and written a test for
 
 Create a file called `hello_test.go`, and write a test called TestHello, just like in previous exercises. Have it call the `Greeting()` function, and expect the result `"Howdy, Gopherinos!"`.
 
+// Why is the test function exported?
```

Comment:
Because `go test` only runs functions of the form `TestXXX`: https://golang.org/pkg/testing/

Reply:
ahhh, is this why the package names are also different so you don't expose the test functions as well?

Reply:
Yes, it's a good idea to always test from _outside_ the package. One reason is that you get to be the first user of your package's API! If it doesn't make sense in the test, it probably won't make sense to other users.

Sometimes there really are things that you can only test from inside, but if we've designed the package well, there shouldn't be very many of these.

Reply:
Nice! I'll make sure to do that when I attempt a project